### PR TITLE
Add a section for Vulnerable Android applications

### PR DESCRIPTION
**Pull request by HrushikeshK**
Add a section to add a list of vulnerable android applications for learning purpose

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -189,6 +189,10 @@ A collection of android security related resources.
 15. ~~[Opcodes table for quick reference](http://ww38.xchg.info/corkami/opcodes_tables.pdf)~~
 15. ~~[APK-Downloader](http://codekiem.com/2012/02/24/apk-downloader/)~~ - seems dead now
 
+### Vulnerable Applications
```
Comment (ashishb) [inline README.md:192]: Can you rename this to "Vulnerable Applications for practice"?

File: README.md [hunk 2]
```diff
@@ -189,6 +189,10 @@ A collection of android security related resources.
...
+1. [Damn Insecure Vulnerable Application (DIVA)](https://github.com/payatu/diva-android)
```
Comment (ashishb) [inline README.md:193]: There are a few other apps like GoatDroid in the previous section, it might be good to move them into this section as well.

**Comment by ashishb** [comment]
Thanks for the contribution.

**Comment by HrushikeshK** [comment]
I have made the necessary changes

**Pull request merged**